sdcm/sct_events/base.py

```diff
@@ -311,7 +311,20 @@ def add_info(self: T_log_event, node, line: str, line_number: int) -> T_log_even
         """
 
         try:
-            self.timestamp = dateutil.parser.parse(line.split(None, 1)[0]).timestamp()
+            splitted_line = line.split(None)
```

Comment:
None is a default argument value... no need to mention it

Reply:
done